packages/vuetify/src/composables/group.ts

```diff
@@ -146,29 +185,40 @@ export function useGroup (
     }
   }
 
-  function getOffsetId (offset: number) {
+  function step (offset: number) {
```

Comment:
What about step reverse? Thinking within the context of v-carousel

Reply:
Not sure what you mean? There are prev and next functions